src/myprofile-page/CurriculumVitae.js

```diff
@@ -0,0 +1,196 @@
+
+const CurriculumVitae = ({
+  actualUser, onUpdate, isUpdating, updateStatus,
+}) => {
+  const [isEditable, setIsEditable] = useState(false);
+  const [addExperience, setAddExperience] = useState(false);
+  const [addEducation, setAddEducation] = useState(false);
+  const [addSkill, setAddSkill] = useState(false);
+
+  const saveEducation = (education) => {
+    onUpdate({
+      ...actualUser,
+      userEducations: [...actualUser.userEducations, education],
+    });
+    isUpdating({ ...updateStatus, education: true });
+    setAddEducation(false);
+  };
+
+  const saveExperience = (experience) => {
+    onUpdate({
+      ...actualUser,
+      userWorkExperiences: [...actualUser.userWorkExperiences, experience],
+    });
+    setAddExperience(false);
+    isUpdating({ ...updateStatus, experience: true });
+  };
+
+  const saveSkill = (skill) => {
+    onUpdate({
+      ...actualUser,
+      userSkills: [...actualUser.userSkills, skill],
+    });
+    setAddSkill(false);
+    isUpdating({ ...updateStatus, skills: true });
+  };
+
+  const renderContent = () => (
+    <Card className="cv-card">
+      <CardContent className="cv-card-content">
+
+        <Typography color="primary" variant="h4">
+          Curriculum Vitae
+        </Typography>
+
+        <br />
+
+        <Typography color="textSecondary" variant="h4">
+          Education
+        </Typography>
+
+        {(actualUser.userEducations || []).map((education) => (
+          <Typography
+            key={education.id}
+            color="textPrimary"
+            variant="body2"
+            component="div"
+            className="bordered-section"
+          >
+            <p>
+              From
+              {' '}
+              {new Date(education.startDate * 1000).toLocaleDateString()}
+              {' '}
+              to
+              {' '}
+              {new Date(education.endDate * 1000).toLocaleDateString()}
+            </p>
+            <h5>{education.institution}</h5>
+            <p>{education.description}</p>
+          </Typography>
+        ))}
+
+        {isEditable && !addEducation
+          ? <IconButton onClick={() => setAddEducation(true)}><Add /></IconButton>
+          : null}
+
+        {addEducation
+          ? (
+            <NewExperienceEntry
+              onSave={saveEducation}
+              id={actualUser.userEducations.length}
+              typename="UserEducation"
+            />
+          )
+          : null}
+
+        <hr />
+
+        <Typography color="textSecondary" variant="h4">
+          Work Experience
+        </Typography>
+
+        {(actualUser.userWorkExperiences || []).map((experience) => (
+          <Typography
+            key={experience.id}
+            color="textPrimary"
+            variant="body2"
+            component="div"
+            className="bordered-section"
+          >
+            <p>
+              From
+              {' '}
+              {new Date(experience.startDate * 1000).toLocaleDateString()}
```

Comment:
This is repeated 4 times in this file, try to extract and reuse it

Reply:
created local function
`const formattedDate = (date) => Moment(new Date(date) * 1000).format(dateTimeFormat);`

Reply:
^ that was for the situation in `CreateMutationHelper.js`, updated on `CurriculumVitae.js` as well with local function
`const getDateFromTimestamp = (date) => new Date(date * 1000).toLocaleDateString();`